### PR TITLE
DEVX-3361 Duplicate Titles

### PR DESCRIPTION
**Pull request by benhayehudi**
## Description

This fixes issue raised in DEVX-3361 where duplicate titles were reported in the `<head>` area.

### CODE REVIEW

File: lib/nexmo_developer/app/presenters/page_title.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class PageTitle
-  def initialize(product, document_title, default_title)
+  CONFIG = YAML.safe_load(File.open("#{Rails.configuration.docs_base_path}/config/header_meta.yml"))
```
Comment (fabianrbz) [inline lib/nexmo_developer/app/presenters/page_title.rb:2]: @benhayehudi can you add a method that returns the title from the config and remove the constant? 
Also, drop the new argument from the initializer

Comment (benhayehudi) [reply lib/nexmo_developer/app/presenters/page_title.rb:2]: ✅

File: lib/nexmo_developer/app/presenters/page_title.rb [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
-      "#{@document_title} | #{DEFAULT}"
+      "#{@document_title} | #{default_title}"
     else
-      DEFAULT
+      default_title
     end
   end
+
+  def default_title
+    config = YAML.safe_load(File.open("#{Rails.configuration.docs_base_path}/config/header_meta.yml"))
```
Comment (fabianrbz) [inline lib/nexmo_developer/app/presenters/page_title.rb:18]: @benhayehudi  I can't make the indention to work in a suggestion :(
but can you memoize the value here?

```
  @default_title ||= begin
    config = YAML.safe_load(File.open("#{Rails.configuration.docs_base_path}/config/header_meta.yml"))
    config['title']
  end
```

Comment (benhayehudi) [reply lib/nexmo_developer/app/presenters/page_title.rb]: done 👍

**Pull request merged**